latex_cite_completions.py

```diff
@@ -271,6 +271,22 @@ def _find_bib_files():
     return cache.LocalCache(root).cache('bib_files', _find_bib_files)
 
 
+def find_open_bib_files(active_window_only=False):
```

Comment:
I'd prefer that the default option was to search the active window only since its possible to have multiple unrelated TeX documents open at the same time.